### ISSUE TITLE
Schedule image doesn't render correctly on Firefox

### ISSUE THREAD

**Issue by captn3m0** [nodejs/lts-schedule#14]

**Comment by AugustinMauroy** [comment]
I was not able to reproduce it. Is it still relevant?

**Comment by captn3m0** [comment]
Yes, this is the rendering I see on GitHub at https://github.com/nodejs/release#release-schedule

![image](https://user-images.githubusercontent.com/584253/223934845-56e90c6a-9947-406d-87e4-eef19c6e012f.png)

Opening in its own tab has the same result:

![image](https://user-images.githubusercontent.com/584253/223934894-e8fc63f3-2d9f-441f-989d-90d1a9dfa09b.png)
